requirements/requirements.in

```diff
@@ -1,5 +1,6 @@
 click
 numpy
+netCDF4
```

Comment:
I had to change this. @swester to be sure it would be good if you test the installation (install from scratch and run one of your verifications).

Reply:
Claire & me both checked our installations, netCDF4 is not listed in the conda-env. Are you sure we need it?

Reply:
Can you try to run the same command as I did here: https://github.com/MeteoSwiss-APN/fls_sat_verif/issues/5  ? (With  
different wd)   If you do not get the error, I will do a new installation without netCDF4 and try again.

Reply:
If you run same command this is a check if the data leads to the problem (likely not). My guess is that in your case for whatever reason (e.g. different version of conda or different time of installation and therefore slightly different resolution of dependencies) NetCDF4 was installed along with xarray, for me (installing last week) this was not the case and I therefore had to add it explicitly.

Reply:
Ok thanks for the info. This could be an explanation. But still strange that netCDF does not show up in the `conda list`... I deleted my env and I will do a clean installation and test.

Reply:
hmmm interesting if it is not in your lists, maybe another backend is used for reading the netCDF files? would be good to have a simple test script, I will add something that I think should also work for other users.

Reply:
I can reproduce Bas' problem, so yes, netCDF4 needs to be added to the requirements. Maybe we had scipy installed @swester and this also has netCDF reading capabilities? 
I only tested the runtime installation, not the dev installation. But I assume the same problem would occur there.

Reply:
Just checked: I had scipy and now I don't. So one of them is required :)

Reply:
Ok, my suggestion would be to explicitly add NetCDF4 (as in this PR), since it is what we want and also more lightweight compared to scipy.